tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -522,4 +522,67 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
     "",
   )
 
+  check(
+    "extends-class",
+    """
+      |package foo
+      |
+      |class Foo ext@@
```

Comment:
What about cases when we have constructor arguments?

Reply:
Ahh, right, maybe we should allow the previous token be a `)` too (as you commented above)

Reply:
done https://github.com/scalameta/metals/pull/4416/commits/43546a252139ec4196090486dcd7a9c071bc8a7c

Reply:
also fixed not to provide keyword completion for

```scala
foo(x) ext@@
```

https://github.com/scalameta/metals/pull/4416/commits/43546a252139ec4196090486dcd7a9c071bc8a7c